### PR TITLE
Add in nix packaging

### PR DESCRIPTION
**Pull request by AbleTheAbove**
nix or nixOS or what have you

### CODE REVIEW

File: example.hbf
Comment (m1el) [inline example.hbf:1]: Binary file in the PR? Maybe add to `.gitignore`?

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+make CC=clang
+./hbas < example.S > example.hbf
```
Comment (m1el) [inline run.sh]: and missing newline at EOF

**Review by m1el — CHANGES_REQUESTED** [review]
Nix pkg is fine, although I'd appreciate a few minor changes

File: example.hbf
Comment (AbleTheAbove) [reply example.hbf:1]: Whoops, fixed.

File: run.sh (hunk 1 above)
Comment (AbleTheAbove) [reply run.sh]: Fixed.

**Comment by AbleTheAbove** [comment]
> Nix pkg is fine, although I'd appreciate a few minor changes

I will be doing a more major pr.

**Pull request merged**